scripts/ASTImportTest.sh

```diff
@@ -37,10 +37,10 @@ fi
 # $1 name of the file to be exported and imported
 # $2 any files needed to do so that might be in parent directories
 function testImportExportEquivalence {
-    if $SOLC $1 $2 > /dev/null 2>&1
+    if $SOLC "$1" $2 > /dev/null 2>&1
     then
         # save exported json as expected result (silently)
-        $SOLC --combined-json ast,compact-format --pretty-json $1 $2> expected.json 2> /dev/null
+        $SOLC --combined-json ast,compact-format --pretty-json "$1" $2 > expected.json 2> /dev/null
         # import it, and export it again as obtained result (silently)
         $SOLC --import-ast --combined-json ast,compact-format --pretty-json expected.json > obtained.json 2> /dev/null
         if [ $? -ne 0 ]
```

Comment:
Although not in diff, why not quote `$?`?

Reply:
Because it's just an integer return code and is not supposed to contain spaces (unless there's a bug in Bash). In any case, shellcheck does not warn about quoting here.